Probesys/bileto
# imp: Change the status by "isResolved" checkbox when opening a ticket

## Related issue(s)



https://github.com/Probesys/bileto/issues/138

## Changes



- remove the status field from the "new ticket" form
- add a checkbox to mark a ticket as resolved

## How to test manually



- open a ticket without checking the checkbox → check the ticket has the "new" status
- open a ticket by checking the checkbox → check the ticket has the "resolved" status

## Checklist
